Flaky events test

<!--
Thanks for submitting a bug report. Please fill the template below,
otherwise we will not be able to process this bug report.
-->

#### Summary
<!-- Summarize the problem in a few sentences: -->

Event package tests are flaky

#### Steps to Reproduce
<!-- How can we reproduce the problem? -->

1. Run tests a couple of times

<!--
Please upload relevant configuration (as .txt).
If you use the command "ttn-lw-stack config", you can redact sensitive config.
-->

#### What do you see now?
<!--
Please paste terminal output, upload logs (as .txt) or upload screenshots.
-->

```
--- FAIL: TestChannelReceive (0.00s)

    doc.go:66: 

        /home/travis/gopath/src/go.thethings.network/lorawan-stack/pkg/events/handler_test.go:75

        Expected '<nil>' to NOT be nil (but it was)!

    doc.go:66: 

        /home/travis/gopath/src/go.thethings.network/lorawan-stack/pkg/events/handler_test.go:80

        Expected: nil

        Actual:   '&{0xc0000c6010 {evt {413352872 63703796623 <nil>} [] <nil> [] travis-job-4be02c26-1a72-47b8-b0df-f54f3ba21d49 map[] 0xc000422900 {} 0} <nil> }'

FAIL

coverage: 65.7% of statements

FAIL	go.thethings.network/lorawan-stack/pkg/events	0.053s
```

#### What do you want to see instead?
<!-- Please add some examples or mock-ups if applicable. -->

Success

#### Environment
<!--
Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
-->

Travis

#### How do you propose to implement this?
<!-- Please think about how this could be fixed. -->

no idea

#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

no

--- a/pkg/events/handler_test.go
+++ b/pkg/events/handler_test.go
@@ -18,7 +18,6 @@ import (
 	"context"
 	"fmt"
 	"testing"
-	"time"
 
 	"github.com/smartystreets/assertions"
 	"go.thethings.network/lorawan-stack/pkg/events"
@@ -72,12 +71,12 @@ func TestChannelReceive(t *testing.T) {
 
 	ctx, cancel := context.WithCancel(test.Context())
 
-	a.So(eventChan.ReceiveTimeout(time.Millisecond), should.NotBeNil)
+	a.So(eventChan.ReceiveTimeout(test.Delay), should.NotBeNil)
 	a.So(eventChan.ReceiveContext(ctx), should.NotBeNil)
 
 	cancel()
 
-	a.So(eventChan.ReceiveTimeout(time.Millisecond), should.BeNil)
+	a.So(eventChan.ReceiveTimeout(test.Delay), should.BeNil)
 	a.So(eventChan.ReceiveContext(ctx), should.BeNil)
 }
 

